### ISSUE TITLE
Add links to wikispecies, where available

### ISSUE THREAD

**Issue by retog** [factsmission/synospecies#8]
```
SELECT DISTINCT ?sn ?article
WHERE
{
   ?article schema:about ?species ;
			schema:isPartOf <https://species.wikimedia.org/> .
   ?species wdt:P225 ?sn.
}
LIMIT 200
```

**Comment by retog** [comment]
A query like 
```
SELECT * WHERE 
            {
                ?species wdt:P225 "Sadayoshia lipkei".
                ?article schema:about ?species ;
			schema:isPartOf <https://species.wikimedia.org/> .

            }
```

would be more restricted then the current:
```
            DESCRIBE ?item WHERE 
            {
                ?item wdt:P225 "Sadayoshia lipkei"
            }
```

Is this wha we want, shall we limit it to wkispecies links? maybe wikispecies and GBIF?

**Comment by nleanba** [comment]
this is implemented in #68 (merged)
